tests/test_paperqa.py

```diff
@@ -1607,27 +1610,46 @@ def disabled_test_memory():
 
 def test_add_texts():
     llm_config = {"temperature": 0.1, "model": "babbage-02"}
-    docs = Docs(llm_model=OpenAILLMModel(config=llm_config))
+    # want to use keyword embedding because
```

Comment:
Just from my perspective, I am not sure how "keyword embedding" correlates to one wanting to use `SparseEmbeddingModel`

I guess, maybe clarify the comment with info how sparse embedding model is equated with keyword embedding

Reply:
yea - typo. Meant to say sparse. Fixed in https://github.com/whitead/paper-qa/commit/9b2eb78a5fb013e2abda279889bff76567999cda